src/tests/reachability/mod.rs

```diff
@@ -5,4 +5,4 @@ pub mod location_validation;
 pub mod parse_partial_state;
 pub mod parser_output_datatypes;
 pub mod parser_restrictions;
-pub mod partial_state;
+pub mod partial_state;
```

Comment:
add eol to this file, something is complaining

Reply:
fixed